compiler/src/iree/compiler/Dialect/Stream/Transforms/ElideTransferOps.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright 2024 The IREE Authors
```

Comment:
```suggestion
// Copyright 2025 The IREE Authors
```

Use the year the file was created.